### PR TITLE
excludeCss option

### LINKED ISSUE #233: saveSvgAsPng without css styleSheets, modifyCss dosen`t work, who can tell me how to use modifyCss

### PR DESCRIPTION
**Pull request by peteruithoven**
Fixes: https://github.com/exupero/saveSvgAsPng/issues/233

### CODE REVIEW

File: src/saveSvgAsPng.js [hunk 1]
```diff
@@ -231,6 +231,7 @@
       height: h,
       scale = 1,
       responsive = false,
+      excludeCss = false,
```
Comment (exupero) [inline src/saveSvgAsPng.js:236]: You should document this in the readme.

File: src/saveSvgAsPng.js [hunk 2]
```diff
@@ -270,22 +271,30 @@
...
-        defs.appendChild(style);
-        clone.insertBefore(defs, clone.firstChild);
-
-        const outer = document.createElement('div');
+      if (excludeCss) {
+        var outer = document.createElement('div');
         outer.appendChild(clone);
-        const src = outer.innerHTML.replace(/NS\d+:href/gi, 'xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href');
-
+        var src = outer.innerHTML;
```
Comment (exupero) [inline src/saveSvgAsPng.js]: These two `var`s can be `const`, correct? If so, `const` is preferred.

**Comment by exupero** [comment]
Thanks for addressing this.

File: src/saveSvgAsPng.js (hunk 2 above)
Comment (peteruithoven) [reply src/saveSvgAsPng.js]: Yes, definitely. Should be fixed now.

**Comment by peteruithoven** [comment]
Thanks for the super quick review.

**Comment by exupero** [comment]
Published 1.4.16 with this.

**Pull request merged**